### PR TITLE
Sdk improvements

### PR DESCRIPTION
**Pull request by soos3d**
Improvement of Particle Auth SDK page

### CODE REVIEW

File: api-reference/auth/desktop-sdks/web.mdx [hunk 1]
```diff
@@ -178,26 +223,149 @@ const userInfo = connect({
...
+
+### Handling User Information
+
+Once the user is logged in, the `userInfo` object from the `useAuthCore()` hook becomes available. 
+
+This object provides a wealth of information that can enhance the user experience (UX). 
+
+For example, you can use it to display the user's `name` and `avatar`, or access security details such as `has_set_master_password`. This allows you to implement logic that prompts the user to set a master password if they haven’t already, thereby improving security.
+
+<Accordion title="Available Fields in `userInfo`">
```
Comment (soos3d) [inline api-reference/auth/desktop-sdks/web.mdx:267]: This is in an accordion, so it looks clean in the actual page.

File: api-reference/auth/desktop-sdks/web.mdx [hunk 2]
```diff
@@ -178,26 +223,149 @@ const userInfo = connect({
...
+
+- `connected` — Returns a boolean value indicating whether the user is logged in (`true`) or not (`false`). This is useful for managing login logic.
+
+- `connectionStatus` — Returns a string representing the current stage of the user's flow:
+  - `loading`
+  - `connecting`
+  - `connected`
+  - `disconnected`
+
+  You can use `connectionStatus` to manage login logic or to display relevant status information in the UI.
```
Comment (soos3d) [inline api-reference/auth/desktop-sdks/web.mdx:257]: Do we want to add a code snippet here?

File: api-reference/auth/desktop-sdks/web.mdx [hunk 3]
```diff
@@ -28,20 +32,20 @@ yarn add @particle-network/auth-core-modal
 
 ## Configuration
 
-Before interacting with Particle, you'll need to import and configure your master configuration object; this'll be used to not only customize the embedded wallet modal which is optionally shown after logging in, but also will allow you to define required keys and authenticate your project.
-
-Although before we import and configure this object, you'll need to retrieve the aforementioned keys.
+The main configuration is set up by wrapping the application with a master configuration object. This object is essential for authenticating your project and customizing the embedded wallet modal, which can be optionally displayed after login.
```
Comment (TABASCOatw) [inline api-reference/auth/desktop-sdks/web.mdx:35]: Configuration is primarily handled by wrapping an application with Particle Auth's master configuration component; this is essential for authenticating your project and [customizing the optional embedded wallet modal](/guides/configuration/appearance/wallet).

File: api-reference/auth/desktop-sdks/web.mdx [hunk 4]
```diff
@@ -28,20 +32,20 @@ yarn add @particle-network/auth-core-modal
...
 
-Regardless of the Particle Auth SDK you use, you'll have the opportunity to define your `projectId`, `clientKey`, and `appId` to authenticate your instance of Particle Auth. In addition to this, customizations can be applied to the resulting wallet modal.
+No matter which Particle Auth SDK you're using, you'll need to define your `projectId`, `clientKey`, and `appId` within this configuration object to properly authenticate your Particle Auth instance. Additionally, you can apply customizations to the wallet modal for a tailored user experience.
```
Comment (TABASCOatw) [inline api-reference/auth/desktop-sdks/web.mdx:37]: `[you can apply customizations to the wallet modal](/guides/configuration/appearance/wallet)`

File: api-reference/auth/desktop-sdks/web.mdx (hunk 2 above)
Comment (TABASCOatw) [reply api-reference/auth/desktop-sdks/web.mdx:257]: Sure, maybe a useEffect monitoring connection status with these methods

File: api-reference/auth/desktop-sdks/web.mdx (hunk 1 above)
Comment (TABASCOatw) [reply api-reference/auth/desktop-sdks/web.mdx:267]: Also if you assign the result of `connect`, it'll yield the same object.

File: api-reference/auth/desktop-sdks/web.mdx [hunk 5]
```diff
@@ -178,26 +223,149 @@ const userInfo = connect({
...
+  - `connected`
+  - `disconnected`
+
+  You can use `connectionStatus` to manage login logic or to display relevant status information in the UI.
+
+### Handling User Information
+
+Once the user is logged in, the `userInfo` object from the `useAuthCore()` hook becomes available. 
+
+This object provides a wealth of information that can enhance the user experience (UX).
```
Comment (TABASCOatw) [inline api-reference/auth/desktop-sdks/web.mdx:263]: Would remove this line

File: api-reference/auth/desktop-sdks/web.mdx [hunk 6]
```diff
@@ -83,23 +92,27 @@ ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
 ```
 </CodeGroup>
 
+<Tip>Learn how to customize the estetics in the [Customizing Login Modal](/guides/configuration/appearance/auth) and [Customizing Wallet Modal](/guides/configuration/appearance/wallet) pages.</Tip>
```
[the remark this answers is not in the document]
Comment (soos3d) [reply api-reference/auth/desktop-sdks/web.mdx:95]: I thought I fixed the mega typo in aesthetics ahah clearly not

File: api-reference/auth/desktop-sdks/web.mdx (hunk 2 above)
Comment (soos3d) [reply api-reference/auth/desktop-sdks/web.mdx:257]: It seems like we don't need useEffect; `connectionStatus` handles everything automatically. I added a simple code snippet.

**Pull request closed without merging**